plugins/hadoop-authorization-sentry-default/hadoop-authorization-sentry/src/main/java/com/thinkbiganalytics/datalake/authorization/config/SentryConfiguration.java

```diff
@@ -44,37 +46,65 @@
 
     @Bean(name = "hadoopAuthorizationService")
     public HadoopAuthorizationService getAuthorizationService(@Value("${beeline.connection.url}") String connectionURL
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Method has 15 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)